mathics/builtin/box/inout.py

```diff
@@ -134,10 +133,11 @@ def render(format, string, in_text=False):
             return format % encode_tex(string, in_text)
 
         if text.startswith('"') and text.endswith('"'):
-            if show_string_characters:
-                return render(r'\text{"%s"}', text[1:-1], in_text=True)
-            else:
-                return render(r"\text{%s}", text[1:-1], in_text=True)
+            # if show_string_characters:
```

Comment:
Why remove this option? (to say the truth I like it)

Reply:
@TiagoCavalcante, this is because in WMA, in TeXForm you do not have double quotes even if you ask `FullForm` or `InputForm`:

```
In[1]:= "Hola"//InputForm                                                       

Out[1]//InputForm= "Hola"

In[2]:= "Hola"//InputForm//TeXForm                                              

Out[2]//TeXForm= \text{Hola}

In[3]:= "Hola"//FullForm//TeXForm                                               

Out[3]//TeXForm= \text{Hola}
```

To reproduce this behavior in Mathics, we need to discard the value of `show_string_characters`.

Reply:
@TiagoCavalcante,
I think I managed to keep the behavior and the option. Please, have a look when you have time.